CTP/isolation/src/com/navercorp/cubridqa/isolation/Main.java

```diff
@@ -42,18 +44,35 @@ public static void exec(String configFilename) throws Exception {
 		system.setProperty("sun.rmi.transport.connectionTimeout", "10000000");
 
 		Context context = new Context(configFilename);
-		context.setBuildId(CommonUtils.getBuildId(context.getCubridPackageUrl()));
-		context.setBuildBits(CommonUtils.getBuildBits(context.getCubridPackageUrl()));
-
-		System.out.println("Build Id: " + context.getBuildId());
-		System.out.println("Build Bits: " + context.getBuildBits());
-
 		ArrayList<String> envList = context.getEnvList();
 		System.out.println("Available Env: " + envList);
 
 		if (context.getEnvList().size() == 0) {
 			throw new Exception("Not found any environment instance to test on it.");
 		}
+		
+		String buildUrl = context.getCubridPackageUrl();
+		
+		if (buildUrl != null && buildUrl.trim().length() > 0) {
+			context.setBuildId(CommonUtils.getBuildId(context
+					.getCubridPackageUrl()));
+			context.setBuildBits(CommonUtils.getBuildBits(context
+					.getCubridPackageUrl()));
+			context.setRebuildYn(true);
+		}else{
+			String envId = context.getEnvList().get(0);
+			String host = context.getInstanceProperty(envId, "ssh.host");
+			String port = context.getInstanceProperty(envId, "ssh.port");
+			String user = context.getInstanceProperty(envId, "ssh.user");
+			String pwd = context.getInstanceProperty(envId, "ssh.pwd");
+			SSHConnect ssh = new SSHConnect(host, user, port, pwd, "ssh"); 
```

Comment:
Please close the connection of ssh.

In addition, it's better to avoid the execution with twice.